Python/bytecodes.c

```diff
@@ -4304,6 +4304,7 @@ dummy_func(
             EXIT_IF(meth->ml_flags != (METH_FASTCALL|METH_KEYWORDS));
             PyTypeObject *d_type = method->d_common.d_type;
             PyObject *self = PyStackRef_AsPyObjectBorrow(arguments[0]);
+            EXIT_IF(self == NULL);
```

Comment:
`self` can only be `NULL` if `total_args` is zero. In that case are reading junk beyond the end of the array, so we we should exit if `total_args == 0` before reading `arguments[0]`.

Reply:
As far as I can tell this was addressed.